### PR TITLE
Create Model Role, Permission for Entrust Ver01

### PR DESCRIPTION
**Pull request by HaCongThanh**

### CODE REVIEW

**Comment by HaCongThanh** [comment]
ready

File: app/Models/Role.php [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+class Role extends EntrustRole
+{
+    use SoftDeletes;
+
+    /**
+     * [$table description]
+     * @var string
+     */
+    protected $table= 'roles';
```
Comment (BunnyPi04) [inline app/Models/Role.php:16]: convention

File: database/migrations/2018_10_30_154010_entrust_setup_tables.php [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+{
+    /**
+     * Run the migrations.
+     *
+     * @return  void
+     */
+    public function up()
+    {
+        // Create table for storing roles
+        if(!Schema::hasTable('roles')){
```
Comment (BunnyPi04) [inline database/migrations/2018_10_30_154010_entrust_setup_tables.php:15]: convention

File: app/Models/Permission.php [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+class Permission extends EntrustPermission
+{
+    use SoftDeletes;
+
+    /**
+     * [$table description]
+     * @var string
+     */
+    protected $table= 'permissions';
```
Comment (BunnyPi04) [inline app/Models/Permission.php:16]: convention

**Pull request merged**